recipes-support/syslog-ng/syslog-ng_3.%.bbappend

```diff
@@ -28,6 +28,12 @@ do_install:append (){
    install -m 0644 ${WORKDIR}/logrotate.d-auth.conf ${D}${sysconfdir}/logrotate.d/auth.conf
    install -m 0644 ${WORKDIR}/logrotate.d-cron.conf ${D}${sysconfdir}/logrotate.d/cron.conf
    install -m 0644 ${WORKDIR}/logrotate.d-messages.conf ${D}${sysconfdir}/logrotate.d/messages.conf
+
+   # NILRT has a custom conf file for sysvinit systems.
+   # If using sysvinit, update the conf file with the current version.
+   if ${@bb.utils.contains('DISTRO_FEATURES','systemd','false','true',d)}; then
+      sed -i -e "s/@VERSION@/$(echo ${PV} | grep -oE ^[0-9]+.[0-9]+)/g" ${D}${sysconfdir}/${BPN}/${BPN}.conf
```

Comment:
`syslog-ng` expects the version string to be `MAJ.MIN` only. Any suggestions on a better way to extract that from ${PV} are welcome.

Reply:
I don't know of a better way. But if you stick to this method, I'd suggest adding a comment. It took me a minute to figure out why you were doing it, and that there was a .patch version you didn't want.

Reply:
Done.